### PR TITLE
bugfix: ARP flux occurs in the virtual machine with multiple interfac…

### PR DESCRIPTION
**Pull request by cl1111**
…es associated with ER

### CODE REVIEW

File: pkg/agent/datapath/localBridge.go [hunk 1]
```diff
@@ -189,7 +189,12 @@ func (l *LocalBridge) cleanLocalIPAddressCache(timeout int) {
 }
 
 func (l *LocalBridge) processLocalEndpointUpdate(arpIn protocol.ARP, inPort uint32) {
-	if !l.isOfPortExists(inPort) {
+	endpoint, isExist := l.getEndpointByPort(inPort)
+	if !isExist {
+		return
+	}
+
+	if endpoint.MacAddrStr != arpIn.HWSrc.String() && endpoint.IPAddr != nil {
```
Comment (KeyongSun) [inline pkg/agent/datapath/localBridge.go:197]: This is incorrect, the endpoint.IPAddr is not the endpoint local IP address, you need to check whether there is local IP for this endpoint. 
Record the endpoint local ip if it has, also need to update the endpoint local ip if the ip address flushed.

File: pkg/agent/datapath/localBridge.go [hunk 2]
```diff
@@ -188,8 +189,53 @@ func (l *LocalBridge) cleanLocalIPAddressCache(timeout int) {
...
+		}
+	}
+}
+
+func (l *LocalBridge) cleanLocalIPAddr(timeout int) {
+	for endpointObj := range l.datapathManager.localEndpointDB.IterBuffered() {
+		endpoint := endpointObj.Val.(*Endpoint)
+		if endpoint.IPAddr == nil {
+			continue
+		}
```
[the remark this answers is not in the document]
Comment (cl1111) [reply pkg/agent/datapath/localBridge.go]: has been modified

**Comment by echkenluo** [comment]
6 个 commit，这不合规范。得本地先 rebase，然后用 force push。 因为你这里就是一个 commit

**Review by KeyongSun — APPROVED** [review]
Please update the test result.

**Comment by cl1111** [comment]
image
image

测试过程：1.嵌套环境下，创建一个VM，包含3个vnic，分别配置IP地址（图中红框所示）；2.ER关联该嵌套集群，创建网络安全策略，策略对象包含该VM，policyBridge里面会增加相应策略的流；3.从策略允许通过的主机ping该VM的一个次接口，观察policyBridge中相关vnic ip的流不会发生变化。

**Pull request merged**